### PR TITLE
#297 - Invalid digest error on blob creation

### LINKED ISSUE #297: Proper error reporting for invalid digest blob
There is error code `DIGEST_INVALID` which should be sent to client when uploaded blob mismatch it's digest

### PR DESCRIPTION
**Pull request by olegmoz**
Closes #297 
Invalid digest error on blob creation

### CODE REVIEW

File: src/test/java/com/artipie/docker/asto/AstoBlobsITCase.java [hunk 1]
```diff
@@ -97,14 +99,22 @@ void failsOnDigestMismatch() {
                     new IsNot<>(new IsNull<>())
                 );
                 MatcherAssert.assertThat(
-                    "Not an IllegalArgumentException exception",
+                    "Not an InvalidDigestException exception",
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/docker/asto/AstoBlobsITCase.java:102]: @olegmoz this message in the assertion seems of, I'd suggest to invert it

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz just one minor comment, also qulice is complaning, please, take a look

**Comment by olegmoz** [comment]
@olenagerasimova thanks, fixed the assertion messages and resolved duplicate literals warning

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks, good to merge

**Pull request merged**